fastlane/Fastfile

```diff
@@ -495,3 +501,29 @@ lane :download_strings_files_from_glotpress do
     download_dir: File.join(PROJECT_FOLDER, 'Simplenote')
   )
 end
+
+# Downloads all the required certificates and profiles for the production build.
+# Optionally, it can create any new necessary certificates or profiles.
+#
+# @option [Boolean] readonly (default: true) Whether to only fetch existing certificates and profiles, without generating new ones.
+lane :app_store_code_signing do |options|
+  # Fail early if secrets not available via `get_required_env`.
+  # Otherwise, Fastlane will prompt to type them.
+  access_key = get_required_env('MATCH_S3_ACCESS_KEY')
+  secret_access_key = get_required_env('MATCH_S3_SECRET_ACCESS_KEY')
+
+  match(
```

Comment:
Nitpick. I prefer to use the action's full name, `sync_code_signing`, because it's clearer to understand what it does when reading the lane. Same for using `run_tests` instead of `scan`, `upload_to_app_store` instead of `deliver`, etc.

What do you think?

---

```
➜ bf action match
[✔] 🚀
Loading documentation for match:

+--------------------------------------------------------------+
|                            match                             |
+--------------------------------------------------------------+
| Alias for the `sync_code_signing` action                     |
```

Reply:
I've gone back and forth on whether to use the "brand" names like `match`, `deliver`, etc. versus the generic names like `sync_code_signing`, etc. Sometimes I forget that the generic name is a Fastlane action and not a helper method in the Fastfile 😅 . But I do agree that the readability of the generic names makes reading through a Fastfile much easier. 

Updated in https://github.com/Automattic/simplenote-macos/commit/462994f3a4026b32ffadf846512fdd73fa7b2fb9